client/app/services/intercom.js

```diff
@@ -1,13 +1,18 @@
 import Ember from 'ember';
 import config from 'client/config/environment';
 
+/* eslint-disable new-cap*/
 export default Ember.Service.extend({
-  intercom: window.Intercom || Ember.K,
+  init() {
+    this._super(...arguments);
+    if (config.intercom.disabled) {
+      window.Intercom = Ember.K;
```

Comment:
Hm. Sounds almost like a load order issue (e.g. this file loading before the Intercom script has a chance to set `window.Intercom`). I wonder if an update to one of our dependencies impacted this.

I think you were on the right track with `this.get('intercom')` – instead of a regular property, we could make `intercom` a computed property on the service that evaluates to either `window.Intercom` or `Ember.K` depending on whether the library is enabled. Then we wouldn't have to override `init`, and also wouldn't have to mutate the global state of `window.Intercom`.